### PR TITLE
chore: update examples to 1.4.0, add example for eager connection config

### PR DESCRIPTION
**Pull request by cprice404**
We have a few customers that need to use the eager connection
configuration to deal with cold start issues.  This commit adds
an example of how to use this configuration.

### CODE REVIEW

File: examples/MomentoApplication/Program.cs [hunk 1]
```diff
@@ -8,97 +8,139 @@
...
-    Console.WriteLine($"\nDeleting cache {CACHE_NAME}");
-    var deleteCacheResponse = await client.DeleteCacheAsync(CACHE_NAME);
-    if (deleteCacheResponse is DeleteCacheResponse.Error deleteCacheError)
+    /// <summary>
+    /// By default, cache clients will connect lazily when the first request is
+    /// issued.  This example shows how to configure a client to connect eagerly
+    /// when it is instantiated, and to specify a distinct timeout for the
+    /// connection to be established.
+    /// </summary>
+    public static async Task EagerConnectionExample()
```
Comment (malandis) [inline examples/MomentoApplication/Program.cs:135]: The GitHub checks complain at this line saying there are no awaited expressions below, so the function should not be async. Does the constructor await the eager connection call?

Comment (nand4011) [reply examples/MomentoApplication/Program.cs:135]: Is there a pre-completed task that can be returned like completedFuture in Java?

Comment (cprice404) [reply examples/MomentoApplication/Program.cs:135]: the constructor call results in a synchronous call to force the connection, so you're right, I don't need async here.

**Review by nand4011 — APPROVED** [review]

File: examples/MomentoApplication/Program.cs (hunk 1 above)
Comment (cprice404) [reply examples/MomentoApplication/Program.cs]: addressed in https://github.com/momentohq/client-sdk-dotnet/pull/363/commits/299c9ad80feb332b48219f4d4afc350ffc436bf9

**Review by malandis — APPROVED** [review]

**Pull request merged**